src/PerfView/EventViewer/EventWindow.xaml

```diff
@@ -17,6 +17,7 @@
     </Window.Resources>
     <DockPanel Background="{DynamicResource BackgroundColour}">
         <DockPanel.CommandBindings>
+            <CommandBinding Command="{x:Static src:EventWindow.ToggleMultiLineViewPaneCommand}" Executed="DoToggleMultiLineViewPane" />
```

Comment:
Would it be possible to make the new window take up the bottom 25% of the window by default?  I think 50% is likely too much.